cmd/nigiri/rpc.go

```diff
@@ -48,7 +48,7 @@ func rpcAction(ctx *cli.Context) error {
 	generate := ctx.Int("generate")
 	named := ctx.Bool("named")
 
-	rpcArgs := []string{"exec", "bitcoin", "bitcoin-cli", "-rpcwallet=" + rpcWallet}
+	rpcArgs := []string{"exec", "bitcoin", "bitcoin-cli", "-rpcuser=admin1", "-rpcpassword=123", "-rpcport=18443", "-rpcwallet=" + rpcWallet}
```

Comment:
not a fan of hardcoding, the idea was to have the bitcoin.conf in datadir and pull rpc creds from there. This behaviro changed in v0.29?